dkms.in

```diff
@@ -619,6 +620,15 @@ read_conf()
     # Set build_exclude
     [[ $BUILD_EXCLUSIVE_KERNEL && ! $1 =~ $BUILD_EXCLUSIVE_KERNEL ]] && build_exclude="yes"
     [[ $BUILD_EXCLUSIVE_ARCH && ! $2 =~ $BUILD_EXCLUSIVE_ARCH ]] && build_exclude="yes"
+    if [[ $BUILD_EXCLUSIVE_CONFIG && -e "$kernel_source_dir/.config" ]]; then
+        local kconf
+        for kconf in $BUILD_EXCLUSIVE_CONFIG ; do
+            case "$kconf" in
+                !*) grep -q -E "^${kconf#!}=[ym]" "$kernel_source_dir/.config" && build_exclude="yes" ;;
+                *)  grep -q -E "^${kconf}=[ym]" "$kernel_source_dir/.config" || build_exclude="yes" ;;
```

Comment:
Please tweak the commit to reuse the "$kernel_config" variable.

Reply:
done